
From: Christoph Hellwig <hch@lst.de>

Two dcache.c functions that shouldn't be used by filesystems directly
(probably a leftover of the intermezzo mess).

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/dcache.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN fs/dcache.c~unexport-is_subdir-and-shrink_dcache_anon fs/dcache.c
--- 25/fs/dcache.c~unexport-is_subdir-and-shrink_dcache_anon	2004-09-26 17:20:37.883656488 -0700
+++ 25-akpm/fs/dcache.c	2004-09-26 17:20:37.887655880 -0700
@@ -1675,8 +1675,6 @@ EXPORT_SYMBOL(dget_locked);
 EXPORT_SYMBOL(dput);
 EXPORT_SYMBOL(find_inode_number);
 EXPORT_SYMBOL(have_submounts);
-EXPORT_SYMBOL(is_subdir);
 EXPORT_SYMBOL(names_cachep);
-EXPORT_SYMBOL(shrink_dcache_anon);
 EXPORT_SYMBOL(shrink_dcache_parent);
 EXPORT_SYMBOL(shrink_dcache_sb);
_
